### PR TITLE
Ocrvs 3662

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -151,7 +147,7 @@ export async function createServer() {
         process.env.NODE_ENV === 'production'
           ? '/client-configs/client-config.prod.js'
           : '/client-configs/client-config.js'
-      console.log("HEY: ",join(__dirname, file))
+      console.log('HEY: ', join(__dirname, file))
```
Comment (euanmillar) [inline src/index.ts]: Sorry @tahmidrahman-dsi this was me probably.  Can you delete it please?

**Review by euanmillar — CHANGES_REQUESTED** [review]
I do not want to delete the shell script.  We should instead remove the print_usage_and_exit function.

I have an idea of how I want to check to ensure a port is open for the countryconfig server before starting like we do in the setup.sh [here](https://github.com/opencrvs/opencrvs-core/blob/9d6a3264359b11919a77b64bef942ff0f2702309/setup.sh#L270).  I want to introduce appropriate port checks to the yarn dev function in core and in country config so keep the shell script for now even if it is very basic.

**Comment by euanmillar** [comment]
@tahmidrahman-dsi @Zangetsu101 please can review my commit here. https://github.com/opencrvs/opencrvs-farajaland/pull/323/commits/4b75bc5c24a2291fb6c9c5950312cff64e6205a6

**Comment by Zangetsu101** [comment]
@euanmillar Looks good although you might have missed this one here: https://github.com/opencrvs/opencrvs-farajaland/blob/21ac67f61a0b12a4ca203fbba2a3098234e34830/docker-compose.countryconfig.prod-deploy.yml#L21-L22
Should be present in the other 2 files as well

**Pull request merged**